.changeset/direct-subconscious-curation.md

```
@@ -0,0 +1,7 @@
+---
+'@mastra/memory': patch
+'@mastra/code-sdk': patch
+'@mastra/factory': patch
+---
+
+Simplify experimental Subconscious knowledge ingestion to run `remind` and `curate` during observation.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)